superset-frontend/src/features/alerts/components/AlertReportCronScheduler.tsx

```diff
@@ -30,18 +29,30 @@ export interface AlertReportCronSchedulerProps {
   onChange: (change: string) => any;
 }
 
+enum ScheduleType {
+  Picker = 'picker',
+  Input = 'input',
+}
+
+type ScheduleTypeOption = `${ScheduleType}`;
+
+const SCHEDULE_TYPE_OPTIONS = [
```

Comment:
@kgabryje Does this file look better/proper? I'm rather new to working with TS and enums.

Reply:
`ScheduleTypeOption` is not necessary. If you change `useState<ScheduleTypeOption>('picker');` to `useState<ScheduleType>(ScheduleType.Picker);` it should work just fine 🙂